v2/internal/util/cel/cel_test.go

```diff
@@ -349,6 +351,32 @@ func Test_CompileAndRunAndCheck(t *testing.T) {
 			expression:  `self.spec.const + self.spec.break + self.spec.__false__ + self.spec.__true__ + self.spec.__in__ + self.spec.__null__ + self.spec.endsWith + self.spec.string + self.spec.int + self.spec.namespace`,
 			expectedStr: "hello! hello! This is a test. I hope it works.",
 		},
+		{
+			name: "two variable comprehension can be used to fix map keys",
+			self: newSimpleResource2Customized(
+				func(r *SimpleResource2) {
+					r.Annotations = map[string]string{
+						"pizza":                   "no",
+						"fruit is good for you":   "yes",
+						"cookies are bad for you": "yes",
+					}
+				}),
+			expression: `self.metadata.annotations.transformMapEntry(k, v, {k.replace(" ", "-"): v})`,
+			expectedMap: map[string]string{
+				"pizza":                   "no",
+				"fruit-is-good-for-you":   "yes",
+				"cookies-are-bad-for-you": "yes",
+			},
+		},
+		{
+			name: "two variable comprehension function addition does not break variable usage",
+			self: newSimpleResource2Customized(
+				func(r *SimpleResource2) {
+					r.Spec.TransformMapField = "test"
```

Comment:
This test is just to make sure that adding the new comprehensions isn't breaking. (It isn't)